### PR TITLE
TIDY Update documentation for debugger

### PR DESCRIPTION
**Pull request by hiemanshu**

### CODE REVIEW

**Comment by hiemanshu** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/recitalsoftware/recital-be-assessment/13?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #10** <a href="https://app.graphite.dev/github/pr/recitalsoftware/recital-be-assessment/10?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #11** <a href="https://app.graphite.dev/github/pr/recitalsoftware/recital-be-assessment/11?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #12** <a href="https://app.graphite.dev/github/pr/recitalsoftware/recital-be-assessment/12?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #13** <a href="https://app.graphite.dev/github/pr/recitalsoftware/recital-be-assessment/13?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/recitalsoftware/recital-be-assessment/13?utm_source=stack-comment).

File: README.md [hunk 1]
```diff
@@ -80,8 +80,15 @@ will not be part of our marking of your solution.
 ### Debugging in Ruby
 
 If you'd like line-by-line debugging, you can drop into a debug session in the
-command line by placing `debugger` as a line of code wherever your want. This
-works as a normal line of code, so you can use conditionals to start a debug
+command line by using the ruby debugger. To get into the debugger add the below code
+
+```
+require "debug"
```
Comment (bmulholland) [inline README.md:86]: You're sure the require is necessary? Shouldn't it be auto-loaded by bundler?

**Review by bmulholland — APPROVED** [review]
Some minor ideas if you want

File: README.md (hunk 1 above)
Comment (hiemanshu) [reply README.md:86]: Yes, it is required. I tested with and without to confirm, and all the aliases don't work without the require.

File: README.md [hunk 2]
```diff
@@ -80,8 +80,15 @@ will not be part of our marking of your solution.
...
+
+binding.break
```
[the remark this answers is not in the document]
Comment (hiemanshu) [reply README.md:88]: Prefer the longer version, more readable, the other aliases `binding.b` and `debugger` are useable too, and listed in the 'debug' gem documentation that we link to. So keeping it simple with the longer naming is my preference

File: README.md (hunk 1 above)
Comment (bmulholland) [reply README.md:86]: Got it, thanks. That's annoying!

**Pull request merged**